scipy/_lib/tests/test_array_api.py

```diff
@@ -120,29 +120,32 @@ def test_check_scalar(self, xp):
 
         # identity always passes
         xp_assert_equal(xp.float64(0), xp.float64(0))
-        xp_assert_equal(xp.array(0.), xp.array(0.))
+        xp_assert_equal(xp.asarray(0.), xp.asarray(0.))
```

Comment:
Of course this is just cosmetic as this only runs with NumPy, but may as well use the functions that are in the standard.